contracts/contracts/coordination/FlatRateFeeModel.sol

```diff
@@ -30,7 +30,7 @@ contract FlatRateFeeModel is IFeeModel {
     }
 
     // TODO: Validate if this is enough to remove griefing attacks
-    function feeDeduction(uint256 pending, uint256 duration) public pure returns (uint256) {
-        return (pending * 1 days) / duration;
+    function feeDeduction(uint256 pending, uint256) public pure returns (uint256) {
+        return pending;
```

Comment:
Should we file a follow-up issue for a post-beta update to this value i.e. to actually allow refunds.

Reply:
opened #229